nginx-controller/controller/controller.go

```diff
@@ -687,6 +687,23 @@ func (lbc *LoadBalancerController) syncIng(task Task) {
 			lbc.recorder.Eventf(ing, api_v1.EventTypeWarning, "Rejected", "%v was rejected: %v", key, err)
 			return
 		}
+
+		if ingEx.MergeableType != "" {
```

Comment:
this should be moved and placed before calling `lbc.createIngress(ing)`, so that either `lbc.validateMergableType(ingEx)` or `lbc.createIngress(ing)` is called, but never both